integration-tests/integration/prisonerProfile/caseNotesPage.spec.js

```diff
@@ -33,7 +33,8 @@ const caseNotes = [
 ]
 
 context('A user can view prisoner case notes', () => {
-  beforeEach(() => {
+  before(() => {
+    cy.clearCookies()
```

Comment:
this is really naff, cypress manual suggests it does this, but it doesn't.  Each test on own succeeds, but group fail without the clear